### PR TITLE
Add guide on deleting the current org

### PR DESCRIPTION
**Pull request by marians**
Follow-up to https://github.com/giantswarm/giantswarm/issues/16564

Adds a guide to the org details page for deleting the org via the Management API

## Preview

image

### CODE REVIEW

**Review by kuosandys — COMMENTED** [review]
Looks good :+1:

File: src/components/MAPI/organizations/guides/DeleteOrganizationGuide.tsx [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+      {...props}
+    >
+      <CLIGuideStepList>
+        <LoginGuideStep />
+        <CLIGuideStep
+          title='2. Delete the organization'
+          command={`kubectl delete organization ${organizationName}`}
+        >
+          <Text>
+            As a result, the organization and the associated namespace will be
```
Comment (axbarsan) [inline src/components/MAPI/organizations/guides/DeleteOrganizationGuide.tsx:53]: Should we also mention that if there are BYOC credentials set up, then the organization will not be deleted until credentials are also deleted?

Comment (marians) [reply src/components/MAPI/organizations/guides/DeleteOrganizationGuide.tsx:53]: That's not the only restriction, is it? Also clusters owned by the org would prevent deletion, right?

**Comment by marians** [comment]
Added a note regarding deletion restrictions and updated the preview screenshot

File: src/components/MAPI/organizations/guides/DeleteOrganizationGuide.tsx [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+        <CLIGuideStep
+          title='2. Delete the organization'
+          command={`kubectl delete organization ${organizationName}`}
+        >
+          <Text>
+            As a result, the organization and the associated namespace including
+            all resources in that namespace will be deleted.
+          </Text>
+          <Text>
+            <strong>Note:</strong> if there are any clusters associated with the
```
Comment (axbarsan) [inline src/components/MAPI/organizations/guides/DeleteOrganizationGuide.tsx:57]: What about provider credentials? If those are set, we prevent a `Organization` CR from being deleted.

File: src/components/MAPI/organizations/guides/DeleteOrganizationGuide.tsx [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+            organization via the label{' '}
+            <code>giantswarm.io/organization={`${organizationName}`}</code>, the
```
Comment (axbarsan) [inline src/components/MAPI/organizations/guides/DeleteOrganizationGuide.tsx:59]:
```suggestion
            <code>giantswarm.io/organization={organizationName}</code>, the
```

File: src/components/MAPI/organizations/guides/DeleteOrganizationGuide.tsx (hunk 2 above)
Comment (marians) [reply src/components/MAPI/organizations/guides/DeleteOrganizationGuide.tsx:57]: I'll add another phrase

File: src/components/MAPI/organizations/guides/DeleteOrganizationGuide.tsx (hunk 3 above)
Comment (marians) [reply src/components/MAPI/organizations/guides/DeleteOrganizationGuide.tsx:59]: Done

**Review by axbarsan — APPROVED** [review]

**Pull request merged**